docs/release_notes.md

```diff
@@ -10,16 +10,19 @@ This release represents a significant effort to make iRODS what users expect fro
-- **Build against libstdc++** - Building against libstdc++ reduces the need to build so many externals packages. As a result, some packages have been migrated to system-provided packages.
+- **Access Time Tracking** - The server has been updated to track access time for all replicas. Updates to the access time are applied in the background to maintain performance. Access time information can be queried using GenQuery.
+- **Semantic Versioning** - Due to significant improvements to the buildsystem, the iRODS 5 series will honor semantic versioning to communicate changes in the software.
+- **Built against libstdc++** - The server is built against libstdc++ by default. As a result, some packages have been migrated to system-provided packages.
 - **GenQuery1** - A brand new flex-bison parser has been added for GenQuery1. This new parser resolves all long-standing bug reports involving embedded single quotes, special characters, reserved keywords, etc.
-- **GenQuery2** - Users have been granted full control over the placement of the `DISTINCT` keyword. This is a major departure from GenQuery1. Unlink GenQuery1, GenQuery2 makes queries explicit and easier to reason about.
-- **Delay Rule Locking** - Migration of a Delay server has been simplified by allowing the Delay server to lock queued delay rules before execution. Information about locked delay rules can be retrieved from the catalog using GenQuery. This new capability opens the door for allowing multiple Delay servers to run within a single zone.
+- **GenQuery2** - The GenQuery2 parser has been updated so that users have full control over the placement of the `DISTINCT` keyword. This is a major departure from GenQuery1. This change is motivated by a need for more control over queries which involve counting.
+- **Delay Rule Locking** - Migration of the Delay server has been simplified by allowing the Delay server to lock queued delay rules before execution. Information about locked delay rules can be retrieved from the catalog using GenQuery.
 
-This release consists of [953 commits from 16 contributors](https://github.com/irods/irods/compare/4.3.0...5.0.0) and [closed 176 issues marked for 5.0.0](https://github.com/irods/irods/issues?q=milestone%3A5.0.0) and [an additional 23 closed issues to be included in the upcoming 4.3.5 release](https://github.com/irods/irods/issues?q=is%3Aissue%20state%3Aclosed%20milestone%3A4.3.5%20closed%3A%3C%3D2025-05-13).
+Unless otherwise noted, the iRODS 5 series will be backward compatible with the iRODS 4.3 series.
+
+This release consists of [957 commits from 16 contributors](https://github.com/irods/irods/compare/4.3.0...5.0.0) and [closed 176 issues marked for 5.0.0](https://github.com/irods/irods/issues?q=milestone%3A5.0.0) and [an additional 23 closed issues to be included in the upcoming 4.3.5 release](https://github.com/irods/irods/issues?q=is%3Aissue%20state%3Aclosed%20milestone%3A4.3.5%20closed%3A%3C%3D2025-05-13).
```

Comment:
let's use 5/20 in the last link on the line

Reply:
Done.